scripts/plot_histogram.py

```diff
@@ -10,16 +10,23 @@
 parser = argparse.ArgumentParser(description=__doc__)
 parser.add_argument("file", help="JSON file with benchmark results")
 parser.add_argument("--title", help="Plot title")
+parser.add_argument(
+    "--labels", help="Comma-separated list of entries for the plot legend"
+)
 parser.add_argument("--bins", help="Number of bins (default: auto)")
 parser.add_argument(
     "--type", help="Type of histogram (*bar*, barstacked, step, stepfilled)"
 )
+parser.add_argument("--imgname", help="Save image to the given filename.")
```

Comment:
I would prefer if we could name this `--output` and maybe provide a `-o` shortcut. That would be in analogy to programs like `curl` or `gcc`.

Reply:
Completely makes sense.